custom/icds_reports/migrations/sql_templates/create_functions.sql

```diff
@@ -2146,3 +2146,74 @@ BEGIN
+        '"child_list".sex, ' ||
+        'child_health_monthly.month, ' ||
+        'child_health_monthly.age_in_months, ' ||
+        'child_health_monthly.open_in_month, ' ||
+        'child_health_monthly.valid_in_month, ' ||
+        'child_health_monthly.nutrition_status_last_recorded, ' ||
+        'child_health_monthly.current_month_nutrition_status, ' ||
+        'child_health_monthly.pse_days_attended, ' ||
+        'child_health_monthly.recorded_weight, ' ||
+        'child_health_monthly.recorded_height, ' ||
+        'child_health_monthly.current_month_stunting, ' ||
+        'child_health_monthly.current_month_wasting, ' ||
+        'child_health_monthly.thr_eligible, ' ||
+        'GREATEST(child_health_monthly.fully_immunized_on_time, child_health_monthly.fully_immunized_late) AS fully_immunized, ' ||
+        'CASE WHEN child_health_monthly.current_month_nutrition_status = ' || quote_literal('severely_underweight') || ' THEN 1' ||
+            'WHEN child_health_monthly.current_month_nutrition_status = ' || quote_literal('moderately_underweight') || ' THEN 2' ||
+            'WHEN child_health_monthly.current_month_nutrition_status = ' || quote_literal('normal') || ' THEN 3' ||
+            'ELSE 4 END AS current_month_nutrition_status_sort,' ||
+        'CASE WHEN child_health_monthly.current_month_stunting = ' || quote_literal('severe') || ' THEN 1' ||
+            'WHEN child_health_monthly.current_month_stunting = ' || quote_literal('moderate') || ' THEN 2' ||
+            'WHEN child_health_monthly.current_month_stunting = ' || quote_literal('normal') || ' THEN 3' ||
+            'ELSE 4 END AS current_month_stunting_sort,' ||
+        'CASE WHEN child_health_monthly.current_month_wasting = ' || quote_literal('severe') || ' THEN 1' ||
+            'WHEN child_health_monthly.current_month_wasting = ' || quote_literal('moderate') || '  THEN 2' ||
+            'WHEN child_health_monthly.current_month_wasting = ' || quote_literal('normal') || ' THEN 3' ||
+            'ELSE 4 END AS current_month_wasting_sort,' ||
+        'CASE ' ||
+          'WHEN child_health_monthly.zscore_grading_hfa = 1 AND child_health_monthly.zscore_grading_hfa_recorded_in_month = 1 THEN ' || quote_literal('severe') || ' ' ||
+          'WHEN child_health_monthly.zscore_grading_hfa = 2 AND child_health_monthly.zscore_grading_hfa_recorded_in_month = 1 THEN ' || quote_literal('moderate') || ' ' ||
+          'WHEN child_health_monthly.zscore_grading_hfa = 3 AND child_health_monthly.zscore_grading_hfa_recorded_in_month = 1 THEN ' || quote_literal('normal') || ' ' ||
+          'ELSE ' || quote_literal('') || ' END AS current_month_stunting_v2, ' ||
+        'CASE ' ||
+          'WHEN (child_health_monthly.zscore_grading_wfh = 1 AND child_health_monthly.zscore_grading_wfh_recorded_in_month = 1) OR (child_health_monthly.muac_grading = 1 AND child_health_monthly.muac_grading_recorded_in_month = 1) THEN ' || quote_literal('severe') || ' ' ||
```

Comment:
Are the `quote_literal`s needed because of the old code is dependent on these being strings? I wonder if there's a better way to do this